src/css/exercises.css

```diff
@@ -120,28 +122,61 @@
   color: rgba(36, 36, 36, 1);
   overflow: hidden;
   text-overflow: ellipsis;
+  flex-shrink: 0;
+  white-space: nowrap;
+  max-width: 55px;
+  @media screen and (min-width: 768px) {
+    max-width: 45px;
+  }
+  @media screen and (min-width: 1440px) {
+    max-width: 100%;
+  }
 }
 
 .exercises-attributes-value::first-letter {
   text-transform: uppercase;
 }
 
-.exercises-favorite-btn,
-.exercises-btn {
+.exercises-rating-btn,
+.exercises-btn,
+.exercises-delete-btn {
   background-color: transparent;
   border: none;
 }
 
-.exercises-favorite-btn {
+.exercises-rating-btn,
+.exercises-delete-btn {
   height: 18px;
   width: 18px;
-  padding: 2px;
+  padding: 1px;
+  flex-shrink: 0;
 }
 
-.exercises-favorite-btn svg {
+.exercises-delete-btn svg {
+  stroke: #242424;
```

Comment:
Could be variable

Reply:
The same in the whole file

Reply:
Didn't find variables yesterday xD will change tomorrow)